package.json

```diff
@@ -31,36 +30,39 @@
     "url": "https://github.com/edx/frontend-template-application/issues"
   },
   "dependencies": {
-    "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
-    "@edx/frontend-component-footer": "10.1.6",
-    "@edx/frontend-component-header": "^2.4.6",
-    "@edx/frontend-platform": "^1.15.6",
-    "@edx/paragon": "19.6.0",
-    "@fortawesome/fontawesome-svg-core": "1.2.36",
-    "@fortawesome/free-brands-svg-icons": "5.15.4",
-    "@fortawesome/free-regular-svg-icons": "5.15.4",
-    "@fortawesome/free-solid-svg-icons": "5.15.4",
-    "@fortawesome/react-fontawesome": "0.1.16",
-    "@reduxjs/toolkit": "^1.8.1",
-    "core-js": "3.15.2",
+    "@edx/browserslist-config": "^1.2.0",
+    "@edx/frontend-component-footer": "13.1.0",
+    "@edx/frontend-component-header": "^5.0.2",
+    "@edx/frontend-platform": "^8.0.0",
+    "@fortawesome/fontawesome-svg-core": "6.5.1",
+    "@fortawesome/free-brands-svg-icons": "6.5.1",
+    "@fortawesome/free-regular-svg-icons": "6.5.1",
+    "@fortawesome/free-solid-svg-icons": "6.5.1",
+    "@fortawesome/react-fontawesome": "0.2.0",
+    "@openedx/brand-openedx": "^1.2.3",
+    "@openedx/frontend-build": "^14.0.3",
```

Comment:
This dependency update will also need to be reverted - although the [package has officially moved](https://www.npmjs.com/package/@edx/frontend-build), the Tutor MFE plugin hasn't been updated to use it, and still uses the [edx package](https://github.com/overhangio/tutor-mfe/blob/15a0c6b2952361b59679dd60479d54a1ce689d2d/tutormfe/templates/mfe/apps/mfe/webpack.dev-tutor.config.js#L7), which causes the MFE container to fail to start in dev.